x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/formula/editor/formula_help.tsx

```diff
@@ -325,12 +332,63 @@ Math functions can take positional arguments, like pow(count(), 3) is the same a
 Use the symbols +, -, /, and * to perform basic math.
                   `,
                   description:
-                    'Text is in markdown. Do not translate function names or field names like sum(bytes)',
+                    'Text is in markdown. Do not translate function names, special characters, or field names like sum(bytes)',
+                })}
+              />
+            </section>
+
+            <section
+              className="lnsFormula__docsTextGroup"
+              ref={(el) => {
+                if (el) {
+                  scrollTargets.current[helpGroups[1].label] = el;
+                }
+              }}
+            >
+              <Markdown
+                markdown={i18n.translate('xpack.lens.formulaCommonFormulaDocumentation', {
+                  defaultMessage: `
+## Common formulas
+
+The most common formulas are dividing two values to produce a percent.
+To display accurately, set *Value format* to *Percent*.
+
+### Filter ratio:
+
+Use \`kql=''\` to filter one set of documents and compare it to other documents within the same grouping.
+For example, to see how the error rate changes over time:
+
+\`\`\`
+count(kql='response.status_code > 400') / count()
+\`\`\`
+
+### Week over week:
```

Comment:
Not a blocker, but h3 has a larger margin-bottom than the code block, which leads to awkward spacing - the next heading being closer to the previous code block than the next paragraph:
<img width="320" alt="Screenshot 2021-06-24 at 10 02 05" src="https://user-images.githubusercontent.com/1508364/123225786-57bb1b00-d4d3-11eb-96fc-0c4bd192ad7e.png">

Compare that to how different functions are spaced:
<img width="249" alt="Screenshot 2021-06-24 at 10 02 10" src="https://user-images.githubusercontent.com/1508364/123225872-6dc8db80-d4d3-11eb-91ee-f88b697ce053.png">

This is done by adding
```
<article className="lnsFormula__docsTextItem">
```

wrapper elements around the individual sections. Maybe we can do the same thing here?